lambdas/main.go

```diff
@@ -423,8 +424,9 @@ func scaleDownHandler(ctx context.Context, info protocol.HostGroupInfoResponse)
 	return scale_down.ScaleDown(ctx, info)
 }
 
-func managementHandler(ctx context.Context, req management.ManagementRequest) (protocol.WekaStatus, error) {
-	switch req.Type {
+func managementHandler(ctx context.Context, req weka_api.WekaApiRequest) (interface{}, error) {
+	fmt.Println("Params %v", req.Params)
+	switch req.Method {
 	case "status":
```

Comment:
this should be the same now for all the weka pi requests, no need for the switch/case

Reply:
4d60cca8682b53101f94f0e629716b150453599e